### PR TITLE
Initial Docker build contribution

### PR DESCRIPTION
**Pull request by mwklein**
Created an independent docker build to allow for execution and testing without using Jenkins. This will also enable experimentation with other build tools. 

Signed-off-by: Michael Klein <michael.w.klein@accenture.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,16 @@ For Corda, we build Docker containers from the Corda source. A number of differe
 
 ![Blockchain Automation Framework - Corda](./docs/images/blockchain-automation-framework-corda.png "Blockchain Automation Framework for Corda")
 
+## Build
+If you are not using the provided Jeknins automation scripts, you can run the provisioning scripts within a docker runtime idependent from your target Kubernetes cluster.
```
Comment (tkuhrt) [inline README.md]: idependent -> independent

File: README.md [hunk 2]
```diff
@@ -28,6 +28,16 @@ For Corda, we build Docker containers from the Corda source. A number of differe
 
 ![Blockchain Automation Framework - Corda](./docs/images/blockchain-automation-framework-corda.png "Blockchain Automation Framework for Corda")
 
+## Build
```
Comment (tkuhrt) [inline README.md]: Is this section specifically related to Corda? If so, this should probably be ### heading. If not, do we need to update line 38, which seems Corda specific?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# USAGE: 
+# docker build . -t baf-build
+# docker run --volume=$(pwd)/network-cordav2.yaml:/home/build/network.yaml baf-build
```
Comment (tkuhrt) [inline Dockerfile]: Is this only valid for Corda?

**Comment by mwklein** [comment]
This contribution is not Corda specific, but I can make that more clear. I'll also fix the typo.

**Comment by mwklein** [comment]
I have identified more requirements beyond just the runtimes for ansible. I will continue to work on this and resubmit a pull request when the full package is working.
